src/uwtools/tests/config/test_support.py

```diff
@@ -68,11 +74,13 @@ def test_log_and_error(caplog):
 
 
 def test_represent_namelist():
+    support.add_yaml_representers()
```

Comment:
Here and just below: These tests were passing by accident because some other test just happened to call this function before these tests ran. Given the new-ish parallelism in our unit tests, these tests were now sometimes failing. Since they require `add_yaml_representers()` to have been called, and since there's no harm (I think) in calling them repeatedly (and hopefully concurrently!) I added explicit calls and could no longer trigger a test failure.